src/ioctl.rs

```diff
@@ -8,12 +8,22 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE-BSD-3-Clause file.
 
-//! Macros and wrapper functions for dealing with ioctls.
+//! Macros and functions for working with
+//! [`ioctl`](http://man7.org/linux/man-pages/man2/ioctl.2.html).
 use libc;
 use std::os::raw::{c_int, c_uint, c_ulong, c_void};
 use std::os::unix::io::AsRawFd;
 
-/// Raw macro to declare the expression that calculates an ioctl number
+/// Macro to declare the expression that calculates an ioctl number.
```

Comment:
I'd leave out the `Macro that...` prefix because, in the generated documentation, these are already grouped under the `Macros` category (check `cargo doc --open`).

Generated doc:
```
Macro vmm_sys_util::ioctl_expr
[+] Show declaration
[−]

Macro to declare the expression that calculates an ioctl number.

use vmm_sys_util::ioctl::_IOC_NONE;

const KVMIO: c_uint = 0xAE;
ioctl_expr!(_IOC_NONE, KVMIO, 0x01, 0);
```

```suggestion
/// Expression that calculates an ioctl number.
```

Reply:
Thanks for your reviewing.  I'll update soon.

Reply:
I've amend the changes into commit. Thanks!